english/polar-bear/polar-bear.md

```diff
@@ -0,0 +1 @@
+The polar bear looks hungry. Maybe i should give him this soft delicious marshmallow. Here you go polar bear. But he doesn’t move. Is he color blind? Maybe he doesn’t see white. So after waiting 10 minutes, i decide to just get my picture taken with him. No wait. I’m too far away, let me step of this fence. He grabs my tennis shoe and tries to eat it. He was hungry!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.